reformat.sh

```diff
@@ -2,5 +2,9 @@
 set -x -e
 
 for file in *.gir; do
-	xmlstarlet ed -P -L "$file"
+	xmlstarlet ed -P -L \
+		-d '//_:doc/@line' \
+		-d '//_:doc/@filename' \
+		-d '//_:source-position' \
```

Comment:
Not sure if it only Windows bug, but sometimes I need use `///` for apply path, here is example, second in fix.sh `freetype2-2.0.gir`.
@sdroege or @GuillaumeGomez can you test if `'///_:source-position'` works for linux on redownload?
Now I need replace it locally and sure forgot other places 😢 
It will be good if I can commit version, that works for me.

Reply:
Also works for me, no changes after changing that, running `dl.sh` and then `fix.sh`, apart from an unrelated change (why?)

```diff
diff --git a/WebKit2-4.0.gir b/WebKit2-4.0.gir
index 341ac25..352d257 100644
--- a/WebKit2-4.0.gir
+++ b/WebKit2-4.0.gir
@@ -6432,7 +6432,7 @@ from an untrusted source.</doc>
         
         <return-value>
           <doc xml:space="preserve">the &lt;function&gt;JSGlobalContextRef&lt;/function&gt; for the #WebKitJavascriptResult</doc>
-          <type c:type="JSGlobalContextRef"/>
+          <type c:type="JSGlobalContextRef" name="JavaScriptCore.GlobalContextRef"/>
         </return-value>
         <parameters>
           <instance-parameter name="js_result" transfer-ownership="none">
@@ -6462,7 +6462,7 @@ returned by webkit_javascript_result_get_global_context() to use the &lt;functio
         
         <return-value>
           <doc xml:space="preserve">the &lt;function&gt;JSValueRef&lt;/function&gt; of the #WebKitJavascriptResult</doc>
-          <type c:type="JSValueRef"/>
+          <type c:type="JSValueRef" name="JavaScriptCore.ValueRef"/>
         </return-value>
         <parameters>
           <instance-parameter name="js_result" transfer-ownership="none">
@@ -16249,7 +16249,7 @@ result values of scripts executed with webkit_web_view_run_javascript().</doc>
         <return-value>
           <doc xml:space="preserve">the &lt;function&gt;JSGlobalContextRef&lt;/function&gt; used by @web_view to deserialize
    the result values of scripts.</doc>
-          <type c:type="JSGlobalContextRef"/>
+          <type c:type="JSGlobalContextRef" name="JavaScriptCore.GlobalContextRef"/>
         </return-value>
         <parameters>
           <instance-parameter name="web_view" transfer-ownership="none">
diff --git a/WebKit2WebExtension-4.0.gir b/WebKit2WebExtension-4.0.gir
index c796d18..93ca5cb 100644
--- a/WebKit2WebExtension-4.0.gir
+++ b/WebKit2WebExtension-4.0.gir
@@ -27566,7 +27566,7 @@ in other web processes may.</doc>
         
         <return-value transfer-ownership="none">
           <doc xml:space="preserve">the JavaScript context of @frame for @world</doc>
-          <type c:type="JSGlobalContextRef"/>
+          <type c:type="JSGlobalContextRef" name="JavaScriptCore.GlobalContextRef"/>
         </return-value>
         <parameters>
           <instance-parameter name="frame" transfer-ownership="none">
@@ -27586,7 +27586,7 @@ between the WebKit and JavaScriptCore APIs.</doc>
         
         <return-value transfer-ownership="none">
           <doc xml:space="preserve">the global JavaScript context of @frame</doc>
-          <type c:type="JSGlobalContextRef"/>
+          <type c:type="JSGlobalContextRef" name="JavaScriptCore.GlobalContextRef"/>
         </return-value>
         <parameters>
           <instance-parameter name="frame" transfer-ownership="none">
```

Reply:
It seem one place where I need use `///`

Reply:
No idea when it works, when not :(

Reply:
@sdroege Please check new version

Reply:
What's the difference between that and what does it mean exactly? `///` means levels down in the XML tree and `//` two levels down? My XPath knowledge disappeared some time 10 years ago :)

Reply:
https://www.w3schools.com/xml/xpath_syntax.asp
"// Selects nodes in the document from the current node that match the selection no matter where they are "
Seems no "///" in standard, but it works for me as // in some cases.

Reply:
and `xmlstarlet` seems almost unsupported, so I ignored this problem as it works normally in Linux.

Reply:
Looks good, no changes after calling `dl.sh`

Reply:
Thanks, not only last to decide what we do with gtk4

Reply:
What's the question?